### PR TITLE
Quick port to 2.0

### PR DESCRIPTION
**Pull request by Anoyomouse**
This is a quick and dirty port to 2.0

Warehouses work, but the migration scripts are broken

I'm going to suggest removing them for 2.0 since we can safely assume that everyone has gone far beyond what they fixed, and 2.0 should be "clean"

### CODE REVIEW

File: prototypes/recipe.lua [hunk 1]
```diff
@@ -7,162 +7,162 @@ data:extend({
...
-			{ "steel-plate", 200 },
-			{ "stone-brick", 40 },
-			{ "iron-stick", 85 },
+			{ type = "item", name = "steel-plate", amount = 200 },
+			{ type = "item", name = "stone-brick", amount = 40 },
+			{ type = "item", name = "iron-stick", amount = 85 },
 		},
 		energy_required = 30,
-		result = "warehouse-basic",
+		results = {{type="item", name="warehouse-basic", amount = 10}},
```
Comment (dgw) [inline prototypes/recipe.lua:18]: Patch would give 10 per craft? 😕

```suggestion
		results = {{type="item", name="warehouse-basic", amount = 1}},
```

**Review by dgw — COMMENTED** [review]
If you're testing on 2.0, that's great. I don't have any advanced access, so for the moment I'll trust you 😜

The last migration script is for Warehousing 0.2.0, and we're at 0.5.x (soon 0.6.0 for the base game upgrade). You're probably right that we can drop the migrations at this point, with an appropriate warning in the README, changelog, and mod portal description.

File: prototypes/recipe.lua (hunk 1 above)
Comment (Anoyomouse) [reply prototypes/recipe.lua:18]: Yeah, whoops, quick one to fix

**Review by Anoyomouse — COMMENTED** [review]
Yeah, they gave me Beta access for modding 2.0, so i've been playing a bit, bit mostly atching how the modders have been breaking 2.0

I can delete the migrations and add stuff to the readme later tonight / tomorrow if you don't get there :)

**Comment by Anoyomouse** [comment]
@dgw Tested and ready to ship for 2.0 release on monday :)

**Comment by dgw** [comment]
@Anoyomouse Don't forget to make a GitHub release for the new version :)

**Comment by dgw** [comment]
Oops. 0.6 didn't have a changelog either. Better late than never, it'll be in 0.6.1… 4473793ef37f07a0a6ad513dfe5e64f35df7bad0

**Comment by Anoyomouse** [comment]
Sorry about that, my fault ... wanted to rush out the mod a little bit before the expansion went live

**Pull request merged**